docs/API.md

```diff
@@ -98,11 +98,15 @@ The following datatypes are used throughout this document:
 * TOTP tokens:
     * createTotpToken           : `PUT /totp/:id`
     * totpToken                 : `GET /totp/:id`
-    * deleteTotpToken           : `DEL /totp/:id`
+    * deleteTotpToken           : `DELETE /totp/:id`
     * updateTotpToken           : `POST /totp/:id/update`
 * Recovery codes:
     * replaceRecoveryCodes      : `POST /account/:id/recoveryCodes`
     * consumeRecoveryCode       : `POST /account/:id/recoveryCodes/:code`
+* Recovery keys:
+    * createRecoverykey         : `POST /account/:id/recoveryKeys`
+    * getRecoveryKey            : `GET /account/:id/recoveryKeys/:recoveryKeyId`
+    * deleteRecoveryKey         : `DELETE /account/:id/recoveryKeys/:recoveryKeyId`
```

Comment:
The confusion around "keys" vs "codes" has never been more obvious than when looking at them right next to each other like that....